coalib/bearlib/aspects/Formatting.py

```diff
@@ -0,0 +1,300 @@
+from coalib.bearlib.aspects import Root, Taste
+
+
+@Root.subaspect
+class Formatting:
+    """
+    This aspect checks on the format of your code.
```

Comment:
this description isn't too helpful IMO, how about something like

```
Formatting defines how your code looks like visually.
```

important here: an aspect doesn't check anything, it's just an aspect of source code.